packages/scripts/config/webpack.config.js

```diff
@@ -307,7 +307,8 @@ const scriptConfig = {
 	plugins: [
 		new webpack.DefinePlugin( {
 			// Inject the `SCRIPT_DEBUG` global, used for development features flagging.
-			SCRIPT_DEBUG: ! isProduction,
+			'globalThis.SCRIPT_DEBUG': JSON.stringify( ! isProduction ),
+			SCRIPT_DEBUG: JSON.stringify( ! isProduction ),
```

Comment:
I'm surprised we don't define `IS_GUTENBERG_PLUGIN` or `IS_WORDPRESS_CORE` here. Third parties that depend on packages probably want those.

Reply:
Why would they need it? WordPress Core exposes all WP packages as entry points and they are already processed with these constants replaced. I don't think that plugins or theme authors would ever want to use these constants as they aren't changing the code of the Gutenberg plugin or WordPress Core.

Reply:
That's a good point, but it would make a difference for folks using wp-script to bundle things outside WordPress. That's probably very uncommon.

Reply:
We can add it if that is the case. Both probably should be `false` in that case.